debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -267,7 +267,7 @@ public void getChangesForTables(String databaseName, SqlServerChangeTable[] chan
             queries[idx] = query;
             // If the table was added in the middle of queried buffer we need
             // to adjust from to the first LSN available
-            final Lsn fromLsn = getFromLsn(databaseName, changeTable, intervalFromLsn);
+            final Lsn fromLsn = getFromLsn(databaseName, changeTable, intervalFromLsn, intervalToLsn);
```

Comment:
The fact that `fromLsn` depends on the `intervalToLsn` looks unexpected. Is it really necessary? See https://github.com/sugarcrm/debezium/pull/85#discussion_r702207266.